features/contract-state/GeneralInfo.tsx

```diff
@@ -139,7 +139,7 @@ const GeneralInfo = () => {
         </Status>
 
         <Status>
-          <Label>GCR (collateral / tokens): </Label>
+          <Label>Global Collateral ratio: </Label>
```

Comment:
Capitalize the "R" in ratio, or lowercase the "C" in collateral to be consistent.

Reply:
ye, you right. this needs to be consistent. have updated it